lib/matplotlib/_fontconfig_pattern.py

```diff
@@ -122,50 +98,31 @@ def parse(self, pattern):
         of key/value pairs useful for initializing a
         `.font_manager.FontProperties` object.
         """
-        props = self._properties = {}
         try:
-            self._parser.parseString(pattern)
-        except self.ParseException as e:
+            parse = self._parser.parseString(pattern)
+        except ParseException as e:
             raise ValueError(
                 "Could not parse font string: '%s'\n%s" % (pattern, e)) from e
-
-        self._properties = None
-
         self._parser.resetCache()
-
+        props = {}
+        if "families" in parse:
+            props["family"] = [*map(_family_unescape, parse['families'])]
+        if "size" in parse:
+            props["size"] = parse["sizes"]
```

Comment:
Typo? (At least I do not understand why checking for `"size"` is a requirement to get `"sizes"`.)

Reply:
oops, fixed.